src/Models/ProductModel.php

```diff
@@ -18,4 +19,31 @@ public function __construct($db)
     {
         $this->db = $db;
     }
+
+    public function addProduct(ProductEntityInterface $productEntity)
+    {
+        $array = ["sku"=>$productEntity->getSku(), "name"=>$productEntity->getName(),
+            "price"=>$productEntity->getPrice(), "stockLevel"=>$productEntity->getStockLevel()];
+
+        $query = $this->db->prepare("UPDATE `products`
+                                    SET `sku` = :sku
+                                        `name` = :name
+                                        `price` = :price
+                                        `stockLevel` = :stockLevel
+        ");
+        $result = $query->execute($array);
+        return $result;
+    }
+
+    public function checkProductExists(string $sku): bool
+    {
+        $query = $this->db->prepare("SELECT `id` FROM `products` WHERE `sku` = '$sku'");
+        $result = $query->execute();
+
+        if(mysqli_num_rows($result) == 0) {
+            false;
+        } else {
+            true;
+        }
```

Comment:
currently not returning the result...? Also, can actually just change this to returning the result of the mysql_nums_rows comparison (and maybe change to strict comparison?)
```suggestion
        return mysqli_num_rows($result) !== 0;
```